policy-controller/grpc/src/workload.rs

```diff
@@ -0,0 +1,87 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize, JsonSchema)]
```

Comment:
We can probably drop the `JsonSchema` macro here. We don't really need to generate a JsonSchema for this type. We'll also drop a dependency if we remove it.

On our CRD bindings it's required by kube-rs' derive macro. It's generally helpful only if you want to create the resource definition (since the api server needs to have a schema for the type).